test/Microsoft.AspNet.Mvc.Xml.Test/XmlDataContractSerializerInputFormatterTest.cs

```diff
@@ -513,637 +504,6 @@ public void SetMaxDepth_ThrowsWhenMaxDepthIsBelowOne()
             Assert.Equal(expectedInt, dummyModel.SampleInt);
             Assert.Equal(expectedString, dummyModel.SampleString);
         }
-
-        [Fact]
-        public async Task PostingListOfModels_HasRequiredAttributeValidationErrors()
```

Comment:
These tests are very specific to the feature being removed, so I ripped them out too.